robottelo/ui/sync.py

```diff
@@ -129,7 +129,11 @@ def enable_rh_repos(self, repos_tree):
         :return: None.
 
         """
-        strategy, value = locators['rh.prd_expander']
+        if ostree:
```

Comment:
Seems little bit as workaround. Do we have reason to have specific scenario for ostree only? I mean, tomorrow we will have one more repo type and then we will add one more if condition. Is it possible to make that scenario more generic? Or at least specify what other types we have in that else statement